config/config.toml.example

```diff
@@ -119,6 +119,9 @@ metrics-addr = ""
 # Prometheus client push interval in second, set \"0\" to disable prometheus push.
 metrics-interval = 15
 
+# if enable tidb record db qps
+record-db-qps = true
```

Comment:
It is better to set it to false as default. Because it may make the server starting failure.

Reply:
OK

Reply:
ok